assets/prometheus-k8s/service-monitor-kubelet.yaml

```diff
@@ -70,6 +70,19 @@ spec:
     tlsConfig:
       caFile: /etc/prometheus/configmaps/kubelet-serving-ca-bundle/ca-bundle.crt
       insecureSkipVerify: false
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    honorLabels: true
+    interval: 30s
+    path: /metrics/probes
```

Comment:
let's double check with a cluster-bot that this actually works

Reply:
seems to be working fine
![image](https://user-images.githubusercontent.com/375856/89036845-bcaad700-d33d-11ea-94c3-ebe1373b56dd.png)